tests/meta/workflow/mutant/conftest.py

```diff
@@ -0,0 +1,319 @@
+from typing import Any
+import pytest
+
+from pathlib import Path
+
+from cg.meta.workflow.mutant.metrics_parser.metrics_parser import MetricsParser
+from cg.meta.workflow.mutant.metrics_parser.models import SampleResults
+from cg.meta.workflow.mutant.quality_controller.models import QualityMetrics, SamplesQualityResults
+from cg.meta.workflow.mutant.quality_controller.quality_controller import MutantQualityController
+from cg.meta.workflow.mutant.quality_controller.utils import get_quality_metrics
+from cg.store.models import Case, Sample
+from cg.store.store import Store
+from cg.constants.constants import ControlOptions
+from tests.store_helpers import StoreHelpers
+from tests.mocks.limsmock import LimsSample, LimsUDF, MockLimsAPI
+
+
+@pytest.fixture
+def mutant_store(store: Store, helpers: StoreHelpers) -> Store:
+    # Add mutant application and application_version
+    application = helpers.add_application(
+        store=store, application_tag="VWGDPTR001", target_reads=2000000, percent_reads_guaranteed=1
+    )
+
+    # Add cases
+    case_qc_pass = helpers.add_case(store=store, name="case_qc_pass", internal_id="case_qc_pass")
+    case_qc_fail = helpers.add_case(store=store, name="case_qc_fail", internal_id="case_qc_fail")
+
+    case_qc_fail_with_failing_controls = helpers.add_case(
+        store=store,
+        name="case_qc_fail_with_failing_controls",
+        internal_id="case_qc_fail_with_failing_controls",
+    )
+
+    # Add samples
+    sample_qc_pass = helpers.add_sample(
+        store=store,
+        internal_id="sample_qc_pass",
+        name="23CS503186",
+        control=ControlOptions.EMPTY,
+        reads=861966,
+        application_tag=application.tag,
+    )
+
+    sample_qc_fail = helpers.add_sample(
+        store=store,
+        internal_id="sample_qc_fail",
+        name="23CS102408",
+        control=ControlOptions.EMPTY,
+        reads=438776,
+        application_tag=application.tag,
+    )
+
+    external_negative_control_qc_pass = helpers.add_sample(
+        store=store,
+        internal_id="external_negative_control_qc_pass",
+        name="0PROVSEK",
```

Comment:
anonymize

Reply:
Does not really matter once the code has been checked in